### PR TITLE
`Paywalls`: rename card to footer

### PR DESCRIPTION
**Pull request by joshdholtz**
### Motivation

Rename "card" to "footer" to because that is how we expect developers to use the current implementation of the non-fullscreen modes as a footer to their custom paywalls.

### Description
- Renamed `card` to `footer`
- Renamed `condensedCard` to `condensedFooter`

### CODE REVIEW

**Review by NachoSoto — APPROVED** [review]
I think this makes much more sense 👍🏻 
FYI I changed the tags. If you make this `breaking` that would trigger purchases-ios 5.0 which I don't think you want to 😅 since we haven't shipped paywalls we can make any breaking changes.

**Comment by NachoSoto** [comment]
Hmm not sure about footer. Let's talk about it tomorrow?

**Comment by joshdholtz** [comment]
> Hmm not sure about footer. Let's talk about it tomorrow?

@NachoSoto Was planning on chatting with you tomorrow about this 😊 Had some realizations with Charlie and Dan today about a footer view (and I have a followup PR that coming soon) that makes will hopefully make this change make more sense

File: Tests/RevenueCatUITests/Templates/Template2ViewTests.swift [hunk 1]
```diff
@@ -23,20 +23,20 @@ class Template2ViewTests: BaseSnapshotTest {
         .snapshot(size: Self.fullScreenSize)
     }
 
-    func testCardPaywall() {
+    func testFooterPaywall() {
```
Comment (NachoSoto) [inline Tests/RevenueCatUITests/Templates/Template2ViewTests.swift:26]: Need to update the snapshot names

**Comment by NachoSoto** [comment]
Merging this.

**Pull request merged**